viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or  not genre or not rating:

```

Comment:
Extra space between operators:
```suggestion
    if not title or not genre or not rating:

```